lab-jonathan/server.js

```diff
@@ -0,0 +1,56 @@
+let http = require('http');
+let fs = require('fs');
+let parseBody = require('./lib/parse-body.js');
+let url = require('url');
+let querystring = require('querystring');
+let cowsay = require('cowsay');
+let PORT = process.env.PORT || 3000;
```

Comment:
You assign port a value, but never use it, which is causing your Travic testing to fail immediately.